### DISCUSSION TITLE
can not clone from linux mint but from os x (MAC)

### DISCUSSION THREAD

**Discussion by ghost** [community/community#78022] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I can login from my Mac via a browser and i can clone to my Mac without beeing asked for username or password
and I con login from my Linux Mint via a browser, but wenn I try to clone the same repository that worked on my Mac I'll be asked for username and password and then I get access denied error. I use the same command to clone in both the mac and the linux terminal.
Need help or hint. Searched some time on the net before asking.

Regards
Rainer

**Comment by amaar-mc** [comment]
If you are asking regarding signing into GitHub and cloning repositories, then on your Linux Mint you may not be logged into GitHub at all. Try running: `git config --list` in your CMD interface and go from there. Then, if you have your GitHub login saved somewhere on your Mac, look at the password and email and make sure you are entering it correctly. If you found this helpful, please mark this as an "answer," I hope this works for you!
